File: crate/fixtures/documents/strings.cs
// A comment mentioning "the docs" is prose, not a literal.
class Messages
{
    const string Path = @"C:\Users\test";
    const string Quoted = @"He said ""hi"" loudly";
    const string Spanning = @"first
second";
    const string Plain = "say \"hi\"";
    const char Marker = '"';
    const string Empty = "";

    static string Greet(string name) => $"Hello, {name}!";
}

File: crate/fixtures/documents/strings.py
"""Module docstring.

Two lines, one string.
"""

# A comment mentioning "the docs" is prose, not a literal.


def greet(name):
    '''Greet a user politely.'''
    template = f"Hello, {name}!"
    pattern = r"\d+ items"
    payload = b"bytes"
    both = rb'raw bytes'
    inline = 'single quoted'
    hashed = "a # inside a string"
    empty = ''
    return template, pattern, payload, both, inline, hashed, empty
